packages/ListBox/stories/moreExamples/Colums/Colums.stories.js

```diff
@@ -0,0 +1,5 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import Columns from "./Columns";
+
+storiesOf("ListBox / more examples", module).add("Colums select", () => <Columns />);
```

Comment:
This story name, this file, and its directory all have the same typo: `colums` → `columns`.